iot_verifier/src/purger.rs

```diff
@@ -57,12 +56,7 @@ impl Purger {
     pub async fn from_settings(settings: &Settings) -> Result<Self, NewPurgerError> {
         let pool = settings.database.connect(PURGER_DB_POOL_SIZE).await?;
         let settings = settings.clone();
```

Comment:
rather than keeping all of settings I prefer to pull the values you need out of settings. It makes it clear what things you need and avoids cloning unneeded things

Reply:
bah, that was how i had been doing it and then noticed this pattern being used elsewhere.

Reply:
yeah, no need to follow bad patterns :-)

Reply:
touche